src/topic/TopicMessageQuery.js

```diff
@@ -388,9 +403,10 @@ export default class TopicMessageQuery {
 
                         setTimeout(() => {
```

Comment:
There's still no log that indicates a retry is in progress

Reply:
I believe I also forgot to add logging in Go; logging just isn't second nature to me yet. Fixed here though https://github.com/hashgraph/hedera-sdk-js/pull/411